src/Storages/System/attachInformationSchemaTables.cpp

```diff
@@ -41,13 +41,13 @@ static void createInformationSchemaView(ContextMutablePtr context, IDatabase & d
                                              database.getTableDataPath(ast_create), context, true).second;
 
         database.createTable(context, ast_create.getTable(), view, ast);
-        if (ast_create.getTable() == "tables")
+        if (database.getDatabaseName() == DatabaseCatalog::INFORMATION_SCHEMA)
```

Comment:
But here in the `createInformationSchemaView` it should be always about the information schema.

Reply:
So, we can create both variants unconditionally.

Reply:
okay